plugins/eos/gs-plugin-eos.c

```diff
@@ -39,6 +39,7 @@
 #define EOS_IMAGE_VERSION_ALT_PATH "/"
 
 #define METADATA_SYS_DESKTOP_FILE "EndlessOS::system-desktop-file"
+#define METADATA_REPLACED_BY_DESKTOP_FILE "Endless::ReplacedByDesktopFile"
```

Comment:
Good point, I suspected we might want to keep this consistent. There's a couple of other places where we're using the `Endless::` namespace with the `UpperCamelCase` notation, the first one that comes to mind is, `Endless::HasDiscoveryFeedContent`. Perhaps once we have that feature working again, we'll want to make that key consistent as well.